src/pages/Ask/Ask.tsx

```diff
@@ -1,9 +1,54 @@
 import React from "react";
 
+import BlueButton from "../../Components/BlueButton/BlueButton";
 import Markdown from "../../Components/Markdown/Markdown";
 
+import styles from "./Ask.module.scss";
+
 const Ask: React.FC = () => {
-  return <Markdown />;
+  return (
+    <>
+      <div className={styles.ask}>
+        <div className={styles.header}>
+          <h1>Ask a public question</h1>
+        </div>
+
+        <div className={styles.content}>
+          <div className={styles.title}>
+            <label>Title</label>
+            <p className={styles.tip}>
+              Be specific and imagine you’re asking a question to another person
+            </p>
+            <input
+              max-length="300"
```

Comment:
property 이름 잘못 쓰셨네요:)
```suggestion
              maxLength="300"
```